docker-compose.yml

```diff
@@ -35,20 +35,34 @@ services:
       zookeeper:
         condition: service_healthy
 
-  trademodule-postgres:
-  ## TODO: Tag this image locally
-    image: "postgres:15-alpine"
+  trademodule-postgis:
+    ## TODO: Tag this image locally
+    image: "postgis/postgis:15-3.3-alpine"
     restart: on-failure
     env_file:
       - env/${ENVIRONMENT}/.env_trademodule
     volumes:
-      - "trademodule_postgres:/var/lib/postgresql/data"
+      - "trademodule_postgis:/var/lib/postgresql/data"
```

Comment:
More of a "note to self" - renaming the volume will leave a dangling on Morris when this is deployed.

Created Ops issue to deal with it: https://github.com/ed-pilots-network/operations/issues/13